source/vocab/display.jsonld

```diff
@@ -108,6 +108,12 @@
           "classLensDomain": "Title",
           "showProperties": [ "mainTitle", "title", "subtitle", "titleRemainder", "partNumber", "partName", "hasPart" ]
         },
+        "KeyTitle": {
+          "@id": "KeyTitle-chips",
+          "@type": "fresnel:Lens",
+          "classLensDomain": "KeyTitle",
+          "showProperties": [ "mainTitle", "qualifier" ]
+        },
```

Comment:
Could be better to just add qualifier to Title chips.

Reply:
Qualifier is not really used in the other Title types whereas KeyTitle only uses mainTitle and qualifier. So I think the specificity serves a dual purpose here in also excluding noise.

Reply:
Makes sense. (And I figure qualifier here is also something indicative of a more relevant distinction we might address in the future.)